ports/nrf/mphalport.c

```diff
@@ -28,10 +28,12 @@
 #include <string.h>
 
 #include "mphalport.h"
+#include "py/mpstate.h"
 
-void mp_hal_stdout_tx_strn(const char *str, mp_uint_t len);
 
 #if (MICROPY_PY_BLE_NUS == 0)
+
+#if !defined( NRF52840_XXAA) || ( defined(CFG_HWUART_FOR_SERIAL) && CFG_HWUART_FOR_SERIAL == 1 )
```

Comment:
If you define a default value in mpconfigport.h then no need to check for defined

Reply:
It is temporarily, it is only defined in pca10056, feather52840 won't have it since it does not have jlink to begin with. I am still not sure if we should keep it hmmm

Reply:
I guess it might be ok to remove the define and just have it always on, as you say, the feather doesn't have jlink, the dongle doesn't have jlink and the PCA10056 has a separate USB connector for the nRF USB, so everyone can use the CDC.

Reply:
Ok, but let's remove it later when CDC is tested reliably enough, we still need a easy switch to jlink for comparison for now.